tests/test_mealie.py

```diff
@@ -262,3 +262,81 @@ async def test_shopping_items(
         headers=HEADERS,
         params=params,
     )
+
+
+async def test_add_shopping_item(
+    responses: aioresponses,
+    mealie_client: MealieClient,
+) -> None:
+    """Test adding shopping item."""
+
+    item: dict[str, Any] = {
+        "shopping_list_id": "27edbaab-2ec6-441f-8490-0283ea77585f",
+        "note": "Bread",
+        "position": 0,
+    }
+
+    url = URL(MEALIE_URL).joinpath("api/groups/shopping/items")
+
+    responses.post(
+        url,
```

Comment:
```suggestion
    responses.post(
       f"{MEALIE_URL}/api/groups/shopping/items",
```
The URL thing was only used for the query parameters in the other tests